service/olark.js

```diff
@@ -0,0 +1,56 @@
+var logger = require('../logging/log.js');
+var _ = require('underscore');
+
+function processChat(chat){
+	var responseTime = getTimeToResponse(chat);
+	var totalChatTime = getTotalChatTime(chat);
+
+	// Repeating a search below but realistically this will
+	// loop through probably at most like 5 small items before
+	// finding one. Oh well.
+	var firstReply = _.findWhere(chat.items, { kind: 'MessageToVisitor'});
+	var operatorId = firstReply.operatorId;
+	var operatorEmail = chat.operators[operatorId] ? chat.operators[operatorId].emailAddress : 'NotAvailable';
+	
+	if (responseTime > 0){
+		logger.info('Received Olark chat data', {
```

Comment:
Should we include some kind of chat identifier in case someone wants to understand why something took so long/short?